### ISSUE TITLE
add bubblejail

### ISSUE THREAD

**Issue by RoyalOughtness** [secureblue/secureblue#22]

**Comment by RoyalOughtness** [comment]
In response to: https://github.com/secureblue/secureblue/issues/9#issuecomment-1826169685

> bubblejail is just an extension for apps that dont work as Flatpaks well, everything integrating with the system etc. I see that bubblewrap-suid is useful, and on Arch it worked just fine for Flatpaks

Bubblejail can be included in this image, but it's much lower priority than getting flatpak/bubblewrap-suid since those have a much bigger positive usability impact.

**Comment by RoyalOughtness** [comment]
Bubblejail copr is ready once bubblewrap-suid is merged: 
https://copr.fedorainfracloud.org/coprs/qoijjj/bubblejail/build/6694557/
